### DISCUSSION TITLE
extreme pamplona HELP PLZ <<3

### DISCUSSION THREAD

**Discussion by cherryxxxval** [ruffle-rs/ruffle#9587] [Q&A]
please someone tell me where i can play extreme pamplona. I have downloaded Ruffle. I dont know anything about this stuff i jsut really want to play my fav childhood game. The websites I have seen allow me to play the first level but does not go beyond that.
Welcome to any suggestions -- btw I dont know anything about this site just saw a couple of old post discussing extreme pamplona
hope someone out there can help
xxoo 
thank you

**Comment by n0samu** [comment]
Extreme Pamplona is not playable in Ruffle yet, but will be in the near future when [this code](https://github.com/ruffle-rs/ruffle/pull/9447) is finished and added. For now, you can play the game in Flashpoint: https://bluemaxima.org/flashpoint/

**Comment by n0samu** [comment]
Update: You can now play the game here! https://flasharch.com/en/archive/play/e36aac73914ec8672218317e000615d7

**Comment by CIACIACIACIACIACIA9** [comment]
I have finished that game very good!
